src/update/token_list.py

```diff
@@ -32,7 +32,12 @@ def update_token_list(dune: DuneAPI, token_list: list[str]) -> list[dict[str, st
     # but we really only care that the data has been pushed and updated
     results = dune.fetch(query)
     # This assertion ensures that the token list was successfully updated.
-    assert len(results) == len(token_list), "Query Failed to push token list"
+    print(token_list)
```

Comment:
```suggestion
    print(token_list)
```

Remove accidental print...